README.md

```diff
@@ -6,7 +6,7 @@
 </h1>
 
 [![GoDoc](https://godoc.org/github.com/secrethub/secrethub-cli?status.svg)][godoc]
-[![Travis CI](https://travis-ci.org/secrethub/secrethub-cli.svg?branch=master)][travis-ci]
+[![CircleCI](https://circleci.com/gh/secrethub/secrethub-cli.svg?style=shield)][circle-ci]
```

Comment:
Consider making the text say `build` instead of `circle-ci`. I always look for that badge, **a** CI badge, not necessarily caring whether it uses Circle, Travis, Jenkins or whatever.